### PR TITLE
fix: display messages in case of error when deleting or leaving group conversation (AR-1556)

### PR DESCRIPTION
**Pull request by yamilmedina**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1556" title="AR-1556" target="_blank">StoryAR-1556</a>  Admin deletes a group conversation for everyone in the group
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

Add case handling for error message when deleting a conversation, or leaving the conversation.

### Solutions

Configure `SnackbarHost` component in Scaffold

### Attachments (Optional)

[fix-error-handling.webm](https://user-images.githubusercontent.com/5806454/188678184-160ddfb3-a250-42bd-87aa-e0f0778e0d03.webm)

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [x] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [x] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Review by vitorhugods — APPROVED** [review]
👨🏼‍🍳🤌🏼

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-937/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4351242)(4351242)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4351242)(4351242)-dev-release.apk)

**Review by gongracr — APPROVED** [review]
Tasty 🍓

File: app/src/main/kotlin/com/wire/android/ui/home/conversations/details/GroupConversationDetailsScreen.kt [hunk 1]
```diff
@@ -113,6 +112,12 @@ private fun GroupConversationDetailsContent(
     val deleteGroupDialogState = rememberVisibilityState<GroupDialogState>()
     val leaveGroupDialogState = rememberVisibilityState<GroupDialogState>()
 
+    val context = LocalContext.current
```
Comment (Garzas) [inline app/src/main/kotlin/com/wire/android/ui/home/conversations/details/GroupConversationDetailsScreen.kt:115]: Do we need to move it to child composable? 😄

**Review by Garzas — APPROVED** [review]
Small comment 😄 
Good eye 👁️ 
LGTM 🚀

**Review by trOnk12 — APPROVED** [review]
Great job 👏

File: app/src/main/kotlin/com/wire/android/ui/home/conversations/details/GroupConversationDetailsScreen.kt (hunk 1 above)
Comment (yamilmedina) [reply app/src/main/kotlin/com/wire/android/ui/home/conversations/details/GroupConversationDetailsScreen.kt]: doone, moved to parent component ! :D

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-937/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4357014)(4357014)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4357014)(4357014)-dev-release.apk)

**Pull request merged**